src/cnglob.h

```diff
@@ -86,7 +86,9 @@ typedef unsigned long long ULLI;
 /*---------------------- Windows definitions -------------------------------*/
 #ifdef _WIN32
 #include <windows.h>
+#define CSE_MAX_PATH _MAX_PATH 	// maximum size for char array to store filename in Windows
 #else
+#define CSE_MAX_PATH 4096 		// maximum size for char array to store filename in POSIX system
```

Comment:
- I can change the name from posix and specify "for GCC and CLang"
- We are using `CSE_MAX_PATH` because that was the original MSVC name. I think it should at least have MAX in the name. It could potentially be `CSE_MAX_PATH_NAME` or `CSE_MAX_FILE_NAME`.

Reply:
1. Ubuntu? What about Red Hat? I'm pretty sure all Linux OSs have the same limitation, but is it the same as MacOS?
2. I was referring to the "filename" in the comment. We want it to be "path".

Reply:
So the comment should read `// maximum size for char array to store path name in Linux OSs and MacOS system`

Reply:
Almost. You still haven't answered the first question: do Linux and MacOS have the path length limits?

Hint: No
Hint 2: There are already a preprocessor definition for this on these other systems.